src/vertexAttribute.js

```diff
@@ -21,7 +21,8 @@ vgl.vertexAttributeKeys = {
   'Normal' : 1,
   'TextureCoordinate' : 2,
   'Color' : 3,
-  'CountAttributeIndex' : 4
+  'Scalar': 4,
+  'CountAttributeIndex' : 5
```

Comment:
I have no idea.  This is typical of bad code in vgl -- vgl.vertexAttributeKeys were defined in two source files in conflicting manners.  When I investigated added higher resolution floats (not what I ended on), I thought adding to this key list would be useful and discovered the problem.  I merged the two definitions, trying to resolve the conflict.